vendors/pc/boards/windows/aws_demos/application_code/aws_demo_logging.c

```diff
@@ -478,6 +478,13 @@ static void prvLoggingPrintf( uint8_t usLoggingLevel,
                                   pcFormat,
                                   xArgs );
         }
+        else
```

Comment:
@htibosch Your change of add the "else" looks correct. Thank you for pointing out the using scenario of 
```
void vLoggingPrint( const char * pcFormat )
{
    prvLoggingPrintf( LOG_NONE, NULL, 0, pdFALSE, pcFormat, NULL );
}
```
The previous version did not address the case when args = NULL and only print out a string.

Reply:
No problem.
This function already has a [long history](https://github.com/aws/amazon-freertos/pull/285#issuecomment-461647595) :-)